### DISCUSSION TITLE
Invalid username or password for admin user

### DISCUSSION THREAD

**Discussion by yudiz-Manushi** [keycloak/keycloak#28401] [Archive]
I'm not able to login into keycloak using admin user and password.
I have changed Authentication Flow for First Broker login flow but now able to revert it to back.
Can you please someone help me here?

Thanks
Manushi Mehta

![image](https://github.com/keycloak/keycloak/assets/152952067/0eba0f36-71d3-426b-ab9e-d24f60f60f95)

**Comment by codespearhead** [comment]
This is probably a configuration issue, as explained [here](https://stackoverflow.com/a/75888072).

However, you can still revert the situation by logging in using the Keycloak Admin CLI, which is present in the bin folder of every Keycloak installation.

1. [Download the version of Keycloak that matches the one in your server](https://github.com/keycloak/keycloak/tags):

```bash
DOWNLOADLINK=https://github.com/keycloak/keycloak/releases/download/24.0.2/keycloak-24.0.2.tar.gz
mkdir demo &&
cd demo &&
curl -L $DOWNLOADLINK  | \
tar -xzvf - --strip-components=1 &&
cd ./bin
```

2. Login into your Keycloak server.

```bash
./kcadm.sh config credentials \
--server https://<keycloak_host>:<keycloak_port>/auth \
--realm <realm-name> \
--user <admin-username> \
--password <admin-password>
```

3. Create a new basic flow in your realm:

```bash
./kcadm.sh create authentication/flows -r <realm-name> \
-s alias="My Browser Flow" \
-s providerId=basic-flow \
-s topLevel=true \
-s builtIn=false
```

4. Set which authentication flow should be used for browser-based logins:

```bash
./kcadm.sh update realms/<realm-name> \
-s 'browserFlow="My Browser Flow"'
```

**Comment by yudiz-Manushi** [comment]
@codespearhead I saw there is no such ./kcadm.sh file or directory.
Is there any other way to do this things?
Please find attached image.

![image](https://github.com/keycloak/keycloak/assets/152952067/35b8086d-bdd8-4b84-baff-e543ac5657b0)

Thanks
Manushi Mehta

**Reply by codespearhead** [reply]
Show us the output of `ls -a`, and confirm what version of Keycloak you downloaded.
